source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
+instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
+static factory method in an aggregation pipeline. 
+
+The ``skip()`` method takes an integer that specifies how many documents
+to omit from the beginning of the list of documents returned by your
+query. 
+
+You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
```

Comment:
Could you explain the "2" value in the example? I think it would help the reader understand its usage without having to dive into the API documentation.